### PR TITLE
chore(deps): use latest version of kwctl

### PR DESCRIPTION
**Pull request by flavio**
Install latest version of kwctl: 1.6.0-rc3

This is needed to run e2e tests of context aware policies

### CODE REVIEW

**Comment by flavio** [comment]
Please don't merge it, I just realized I have to tag a new release of kwctl

File: kwctl-installer/action.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ inputs:
   KWCTL_VERSION:
     description: 'kwctl release to be installed'
     required: false
-    default: v1.6.0-rc2
+    default: v1.6.0-rc3
```
Comment (jvanz) [inline kwctl-installer/action.yaml:10]: As far as I can see, this will be updated to `v1.6.0-rc4`. Right?

Comment (flavio) [reply kwctl-installer/action.yaml:10]: yes, I'll update the PR

Comment (flavio) [reply kwctl-installer/action.yaml]: I'm using rc4 now

**Comment by flavio** [comment]
We need to wait for https://github.com/kubewarden/kwctl/actions/runs/4449467150 to be done before merging this PR.

I guess we will have to create a new tag once this is merged

**Pull request merged**